app/exchangers/isagri/isacompta/fixed_assets_exchanger.rb

```diff
@@ -178,6 +101,31 @@ def import
           w.check_point
         end
       end
+
+      private
+
+      # Trim account number following preferences
+      def normalize(number)
+        preference_number = Preference[:account_number_digits]
+        differential = number.size - preference_number if number.size > preference_number
+        if differential && number.last(2) == "0" * (differential)
+          number.first(preference_number)
+        elsif differential && number.last != "0" * (differential)
+          number.first(6) + number.last(preference_number - 6)
+        else
+          number
+        end
+      end
+
+      # Generate allocation account number
+      def to_allocation_account(number)
+        if number.first == "2"
+          number.chars.insert(1, "8")[0...-1].join
+        else
+          number
```

Comment:
Is there a reason why the function defaults to the account number and not just fails when we cant 'convert' automatically to an account number?